cmd/src/campaigns_create.go

```diff
@@ -34,7 +41,7 @@ Examples:
 	}
 	var (
 		nameFlag        = flagSet.String("name", "", "Name of the campaign. (required)")
-		descriptionFlag = flagSet.String("desc", "", "Description for the campaign. (required)")
+		descriptionFlag = flagSet.String("desc", "", "Description for the campaign in Markdown. (required)")
```

Comment:
`-name` and `-description` are no longer required flags, but the flag help says `(required)`